packages/cms-lib/lib/watch.js

```diff
@@ -113,7 +113,10 @@ function watch(portalId, src, dest, { mode, cwd, remove, disableInitial }) {
             );
           });
       });
-    });
+    };
+    
+    watcher.on('unlink', deleteFileOrFolder);
+    watcher.on('unlinkDir', deleteFileOrFolder);
```

Comment:
Thanks @gkemp94!

Code looks good and your branch is running as expected.

One improvement: we can still log a specific statement as to whether a file or a folder was deleted via partial application or a closure.

Something like this should work

```javascript
const deleteFileOrFolder = type => filePath => {
  /*****/
  logger.log('Deleted %s "%s"', type, remotePath);
  /*****/
}

watcher.on('unlink', deleteFileOrFolder('file'));
watcher.on('unlinkDir', deleteFileOrFolder('folder'));
```

Reply:
Great Suggestion! That's been updated!